### PR TITLE
Update H2Network page

### PR DESCRIPTION
**Pull request by gpetit**
As requested in #913 add some words about H2Network (and also add two bibliographic ref)

### CODE REVIEW

File: docs/dev/applications/h2network.md [hunk 1]
```diff
@@ -6,6 +6,31 @@ prev_section: system
...
 ---
 
+
+
+`H2Network` extends the `H2` / `H2GIS` geospatial database to provide graph routing functionalities. 
+
+`H2Network` is based on the [Java Network Analyzer](https://github.com/orbisgis/java-network-analyzer) (`JNA`) library. 
+
+JNA provides a collection of graph theory and social network analysis algorithms. These algorithms are implemented on mathematical graphs using the [JGraphT](https://jgrapht.org/) library.
+
```
Comment (ebocher) [inline docs/dev/applications/h2network.md]: H2GIS offers a collection of SQL functions on top of the JNA.

File: docs/dev/applications/h2network.md [hunk 2]
```diff
@@ -6,6 +6,31 @@ prev_section: system
...
+### Install H2Network
+
+To install `H2Network` execute the two following instructions.
```
Comment (ebocher) [inline docs/dev/applications/h2network.md]: H2Network is delivered with the H2GIS binaries. So to ... just run the following

File: docs/dev/applications/h2network.md [hunk 3]
```diff
@@ -6,6 +6,31 @@ prev_section: system
...
+
+{% highlight mysql %}
+CREATE ALIAS IF NOT EXISTS H2GIS_NETWORK FOR "org.h2gis.network.functions.NetworkFunctions.load";
+CALL H2GIS_NETWORK();
+{% endhighlight %}
+
+
+### H2Network functions
+
 The following H2Network functions are available:
```
Comment (ebocher) [inline docs/dev/applications/h2network.md]: The following SQL functions are available:

File: docs/dev/applications/h2network.md (hunk 2 above)
Comment (gpetit) [reply docs/dev/applications/h2network.md]: ok

File: docs/dev/applications/h2network.md (hunk 1 above)
Comment (gpetit) [reply docs/dev/applications/h2network.md]: ok

File: docs/dev/applications/h2network.md (hunk 3 above)
Comment (gpetit) [reply docs/dev/applications/h2network.md]: ok

**Comment by gpetit** [comment]
@ebocher you can merge if my (v2) modifications are ok for you

**Pull request merged**